### DISCUSSION TITLE
Add Private Subnet to $Home_Net

### DISCUSSION THREAD

**Discussion by ahezza** [Security-Onion-Solutions/securityonion#8560] [Unsupported Versions] [answered]
I have a IPs in subnet 10.0.2.1/24 which are triggering as an External Host. e.g. the Suricata alert below:

ET INFO External Host Probing for ChromeCast Devices
Source = 10.0.0.23
Destination = 10.0.2.6

How do I add 10.0.2.1/24 into the private IP range post install?

**Comment by xfaith** [comment, marked as answer]
Per the document: https://docs.securityonion.net/en/2.3/homenet.html?highlight=HOMENET
Edit the global.sls and add the new IP address. 
Then   sudo salt $SENSORNAME_$ROLE state.apply suricata,zeek
By default, a node will use the global homenet pillar value if it is defined in the global pillar file (/opt/so/saltstack/local/pillar/global.sls) under global:hnmanager.

**Reply by ahezza** [reply]
Many thanks, exactly what I was after.
Opens up a new question why this Suricata Alert is triggering on this alert when the 10.0.0.0/8 is part of Home_Net.. Anyway, I will likely suppress this rule anyway. Cheers.

**Comment by ccamejo1** [comment]
ahezza,

To answer your question on why the alert is triggering if you have your HOME_NET set, take a look at the rule itself.

alert http $EXTERNAL_NET any -> $HOME_NET any (msg:"ET INFO External Host Probing for ChromeCast[?](https://doc.emergingthreats.net/bin/edit/Main/ChromeCast?topicparent=Main.2026758) Devices"; flow:established,to_server; .....

Basically, this is triggering on any $EXTERNAL_NET IP on any port going to your $HOME_NET.  The root of the issue is that the $EXTERNAL_NET variable defaults to 'any'.  It is recommended to change this to anything but your HOME_NET instead of any.   You can find this setting in /opt/so/saltstack/default/salt/suricata/defaults.yaml .  As always copy the file to the local directory tree and make your changes there. (/opt/so/saltstack/local/salt/suricata/default.yaml)

Just set the EXTERNAL_NET: to "!HOME_NET" instead of the default "any".  This will stop Suricata from triggering alerts from your internal defined HOME_NET.

**Reply by ahezza** [reply]
This is great, thank you!

**Comment by dougburks** [comment]
For more information about modifying `HOME_NET`, please see:
https://docs.securityonion.net/en/2.3/homenet.html#homenet

By default, `EXTERNAL_NET` is set to `any` (which includes `HOME_NET`) to detect lateral movement inside your environment. If you would like to change this default, please see:
https://docs.securityonion.net/en/2.3/suricata.html#configuration
